robot-core/src/main/java/org/obolibrary/robot/metrics/MetricsLabels.java

```diff
@@ -0,0 +1,201 @@
+package org.obolibrary.robot.metrics;
+
+import java.lang.reflect.Field;
+import java.util.HashMap;
+import java.util.Map;
+
+@SuppressWarnings("unused")
```

Comment:
Can we remove this now?

Reply:
I don't mind - it annoyed me to see the warnings for unused methods - which I left purely because I think I might need them going forward, but I don't have another place for them anymore. But yeah, feel free to delete!